bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/IndexPersistenceMgr.java

```diff
@@ -267,14 +267,7 @@ private File findIndexFile(long ledgerId) throws IOException {
     }
 
     boolean ledgerExists(long ledgerId) throws IOException {
```

Comment:
the change here seems to change the semantic. it only looks activeLedgers. if it is missing in activeLedgers, it would respond NoLedger exception. this seems to be wrong to me.